<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Effects</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="currentRow">
         <number>-1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Events</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox"/>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget_2"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Actions</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_2">
        <item>
         <property name="text">
          <string>Scale</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rotate</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Move</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fade</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Paint</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Change Color</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Change Shadow Size</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Change Border Size</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Set Texture</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Shake</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Wiggle</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Glow</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Blur</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Start Group</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>End Group</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget_3"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>728</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEffects">
    <property name="title">
     <string>Effects</string>
    </property>
    <addaction name="actionAdd_Effect"/>
    <addaction name="actionRemove_Effect"/>
    <addaction name="actionEdit_Effect"/>
   </widget>
   <widget class="QMenu" name="menuEvents">
    <property name="title">
     <string>Events</string>
    </property>
    <addaction name="actionAdd_Event"/>
    <addaction name="actionRemove_Event"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionAdd_Action"/>
    <addaction name="actionRemove_Action"/>
   </widget>
   <addaction name="menuEffects"/>
   <addaction name="menuEvents"/>
   <addaction name="menuActions"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_Effect"/>
   <addaction name="actionRemove_Effect"/>
   <addaction name="separator"/>
   <addaction name="actionAdd_Event"/>
   <addaction name="actionRemove_Event"/>
   <addaction name="separator"/>
   <addaction name="actionAdd_Action"/>
   <addaction name="actionRemove_Action"/>
   <addaction name="separator"/>
   <addaction name="actionGenerate"/>
   <addaction name="separator"/>
   <addaction name="actionCheckCard"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd_Effect">
   <property name="text">
    <string>Add Effect</string>
   </property>
  </action>
  <action name="actionAdd_Event">
   <property name="text">
    <string>Add Event</string>
   </property>
  </action>
  <action name="actionRemove_Effect">
   <property name="text">
    <string>Remove Effect</string>
   </property>
  </action>
  <action name="actionGenerate">
   <property name="text">
    <string>Generate</string>
   </property>
  </action>
  <action name="actionCheckCard">
   <property name="text">
    <string>Check if your credit card has funds</string>
   </property>
  </action>
  <action name="actionAdd_Action">
   <property name="text">
    <string>Add Action</string>
   </property>
  </action>
  <action name="actionRemove_Event">
   <property name="text">
    <string>Remove Event</string>
   </property>
  </action>
  <action name="actionRemove_Action">
   <property name="text">
    <string>Remove Action</string>
   </property>
  </action>
  <action name="actionEdit_Effect">
   <property name="text">
    <string>Edit Effect</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
